### PR TITLE
added stanford_search_api_exclude_nodes as an optional module to be a…

### PR DESCRIPTION
**Pull request by boznik**
…ble to exclude selected nodes from a solr index

### CODE REVIEW

File: modules/stanford_search_api_exclude_nodes/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+Installation
+---
+
+Install this module like any other module. [See Drupal Documentation](https://drupal.org/documentation/install/modules-themes/modules-7)
+
+Configuration
+---
+
+Nothing special needed.
```
Comment (sherakama) [inline modules/stanford_search_api_exclude_nodes/README.md]: Please leave a note about adding the required field to content types.

**Comment by sherakama** [comment]
@boznik, Just add the configuration instructions and you can merge.

File: modules/stanford_search_api_exclude_nodes/README.md (hunk 1 above)
Comment (boznik) [inline modules/stanford_search_api_exclude_nodes/README.md]: @sherakama you want me to move line 12 under Configuration?

Comment (sherakama) [inline modules/stanford_search_api_exclude_nodes/README.md]: The note above should suffice.

**Pull request merged**